### PR TITLE
SB-25552 - UCI odk deployment

### PR DESCRIPTION
**Pull request by shiva-rakshith**
<table>
<td>
  <a href="https://project-sunbird.atlassian.net/browse/SB-25552" title="SB-25552" target="_blank">StorySB-25552</a>  UCI - Deployment
  </td></table>
  <br />
 

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Enhancement (additive changes to improve performance)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions:
* Hardware versions:

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by shiva-rakshith** [comment]
@keshavprasadms Please review the PR.

**Comment by ChakshuGautam** [comment]
@keshavprasadms Please review

File: kubernetes/helm_charts/core/odk/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        envFrom:
+        - configMapRef:
+            name: {{ .Chart.Name }}-config
+        resources:
+{{ toYaml .Values.resources | indent 10 }}
+        ports:
+        - containerPort: {{ .Values.network.port }}
+        {{- if .Values.healthcheck }}
+        livenessProbe:
+{{ toYaml .Values.livenessProbe | indent 10 }}
```
Comment (keshavprasadms) [inline kubernetes/helm_charts/core/odk/templates/deployment.yaml:43]: Please add the health checks values.

Comment (shiva-rakshith) [reply kubernetes/helm_charts/core/odk/templates/deployment.yaml:43]: Added health check values.

**Review by keshavprasadms — APPROVED** [review]
LGTM

**Pull request merged**